cookiecutter.json

```diff
@@ -10,7 +10,7 @@
   "conda_pypi_package_dist_name": "{{ cookiecutter.project_name|replace(' ', '-')|lower }}",
   "package_dir_name": "{{ cookiecutter.project_name|replace(' ', '_')|replace('-', '_')|lower }}",
   "project_short_description": "Python package for doing science.",
-  "keywords": "diffraction, PDF, X-ray, neutron",
+  "project_keywords": "diffraction, PDF, X-ray, neutron",
```

Comment:
For consistency `project_short_description` and `project_keywords` which are both provided under `pyproject.toml` I've updated the manuscript if it okay.

<img width="658" alt="Screenshot 2025-05-03 at 6 01 23 PM" src="https://github.com/user-attachments/assets/bc2f4e73-c011-4c5a-b6e2-b7200e083e58" />